src/Microsoft.AspNetCore.Server.Kestrel/Internal/Exceptions/BadHttpRequestException.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.IO;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Exceptions
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Exceptions
 {
     public sealed class BadHttpRequestException : IOException
```

Comment:
Could this be in `Microsoft.AspNetCore.Server` and pushed up to a common? Maybe also used by WebListener? /cc @Tratcher